pkg/scheduler/frameworkext/debug_scores_filters.go

```diff
@@ -29,11 +29,13 @@ import (
 )
 
 var (
-	debugTopNScores = 0
+	debugTopNScores    = 0
+	debugFilterFailure = false
 )
 
 func AddFlags(fs *pflag.FlagSet) {
 	fs.IntVarP(&debugTopNScores, "debug-scores", "s", debugTopNScores, "logging topN nodes score and scores for each plugin after running the score extension, disable if set to 0")
+	fs.BoolVarP(&debugFilterFailure, "debug-filters", "f", debugFilterFailure, "logging filter failures")
```

Comment:
Why not reuse `--v` to set the log level?

Reply:
this already exists in cmd/koord-scheduler/app/server.go

func installProfilingHandler(pathRecorderMux *mux.PathRecorderMux, enableContentionProfiling bool) {
	routes.Profiling{}.Install(pathRecorderMux)
	if enableContentionProfiling {
		goruntime.SetBlockProfileRate(1)
	}
	// NOTE: Use utilroutes.DebugFlags instead of k8s.io/apiserver/pkg/server/routes.DebugFlags
	//  as using the latter will print a useless stack when installing multiple flags
	debugFlags := utilroutes.NewDebugFlags(pathRecorderMux)
	debugFlags.Install("v", utilroutes.StringFlagPutHandler(logs.GlogSetter))
	debugFlags.Install("s", utilroutes.StringFlagPutHandler(frameworkext.DebugScoresSetter))
	debugFlags.Install("f", utilroutes.StringFlagPutHandler(frameworkext.DebugFiltersSetter))
}

Reply:
I mean you can reuse the flag instead of adding a new one.

Reply:
If we only want to diagnose the filter failed reasons, we dont't need to much unrelated verbose logs.

Reply:
> Why not reuse `--v` to set the log level?

Most users set the log level to V(4) or even lower, and then decide whether to enable this diagnostic method according to their needs. Most of the time, it is OK to control whether to output this log through the log level, but it is a bit troublesome to only prohibit the output of this log.